.travis.yml

```diff
@@ -31,7 +31,6 @@ script:
   # (see comment where the feature is defined in mp4parse_capi/Cargo.toml),
   # but we can at least check for changes behind features that would break the
   # build.
-  - cargo check --all --verbose $RELEASE --tests --all-features
```

Comment:
We can remove the comment above as well, since it's out of date with this change

Reply:
Good point! Will do.

Reply:
Done